.travis.yml

```diff
@@ -9,29 +9,30 @@ cache:
 
 env:
   global:
+    - MODE="test"
     - NUMPY="1.14"
     - SCIPY="false"
     - COVERAGE="false"
-    - STATIC="false"
-    - DOCS="false"
 
 jobs:
   include:
-    - stage: test
-      env: PYTHON="2.7" STATIC="true"
-    - env: PYTHON="2.7" COVERAGE="true"
-    - env: PYTHON="2.7" NUMPY="1.11"
+    - stage: basic
+      env: MODE="static" PYTHON="2.7"
+    - env: MODE="test" PYTHON="2.7" COVERAGE="true"
+    - stage: advanced
+      env: MODE="test" PYTHON="2.7" NUMPY="1.11"
       cache: false  # disable the cache for one build to make sure that works
-    - env: PYTHON="2.7" NUMPY="1.12"
-    - env: PYTHON="2.7" NUMPY="1.13"
-    - env: PYTHON="2.7" NUMPY="1.14"
-    - env: PYTHON="3.4" NUMPY="1.11"  # <=1.12 not packaged for py34 in conda
-    - env: PYTHON="3.5"
-    - env: PYTHON="3.6" COVERAGE="true" SCIPY="true"
-    - env: PYTHON="3.6" DOCS="true"
+    - env: MODE="test" PYTHON="2.7" NUMPY="1.12"
+    - env: MODE="test" PYTHON="2.7" NUMPY="1.13"
+    - env: MODE="test" PYTHON="2.7" NUMPY="1.14"
+    # <=1.12 not packaged for py34 in conda
+    - env: MODE="test" PYTHON="3.4" NUMPY="1.11"
```

Comment:
This isn't new in this PR, but just noticed it.  Isn't our build here (using `numpy==1.11`) contradicting the comment?

Reply:
Yeah I think this is supposed to say >=, my b